server/src/test/java/com/google/coffeehouse/common/StorageHandlerHelperTest.java

```diff
@@ -0,0 +1,214 @@
+*/
+@RunWith(JUnit4.class)
+public class StorageHandlerHelperTest {
+  static DatabaseClient dbClient;
+  private static final String INSTANCE_ID = "coffeehouse-instance-test";
+  private static final String DATABASE_ID = "coffeehouse-db-test";
+
+  @Before
+  public void setUp() {
+    // Create spanner service and database client
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+    DatabaseId db = DatabaseId.of(options.getProjectId(), INSTANCE_ID, DATABASE_ID);
+    dbClient = spanner.getDatabaseClient(db);
+
+    // Delete all the data while keeping the existing database
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(Mutation.delete("Persons", KeySet.all()));
+    mutations.add(Mutation.delete("Books", KeySet.all()));
+    mutations.add(Mutation.delete("Clubs", KeySet.all()));
+    dbClient.write(mutations);
+  }
+
+  public void insertPerson() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Persons")
+        .set("userId")
+        .to("person")
+        .set("email")
+        .to("person@test.com")
+        .set("nickname")
+        .to("person")
+        .set("pronouns")
+        .to("she/he/they")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public void insertClub() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Clubs")
+        .set("clubId")
+        .to("club")
+        .set("bookId")
+        .to("book")
+        .set("description")
+        .to("A club")
+        .set("name")
+        .to("Club")
+        .set("ownerId")
+        .to("owner")
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public void insertMembership() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertBuilder("Memberships")
+        .set("userId")
+        .to("person")
+        .set("clubId")
+        .to("club")
+        .set("membershipType")
+        .to(MembershipConstants.MEMBER)
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public void insertBook() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("book")
+        .set("author")
+        .to("book")
+        .set("isbn")
+        .to("97800609459470")
+        .set("title")
+        .to("book")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  @Test
+  public void checkPersonInClubQuery_personInClub() throws Exception {
+    insertPerson();
+    insertClub();
+    insertMembership();
+    Boolean actual = StorageHandlerHelper.checkPersonInClubQuery(dbClient, "person", "club");
+    Boolean expected = true;
+    assertEquals(expected, actual);
```

Comment:
nit: No need for extra vars

```suggestion
    assertEquals(true, actual);
```